op-enable.go

```diff
@@ -362,10 +362,8 @@ func getArgs(s DockerHandlerSettings, dd driver.DistroDriver) string {
 func getDockerUrls(env string) (string, string) {
 	urlMap := map[string]map[string]string{
 		"china": map[string]string{
-			// This script is the same as https://get.docker.com, except
-			// 1. apt_url and yum_url points to the mirror in China
-			// 2. fix an issue in detecting distro versions
```

Comment:
Pulled a diff:

```
diff --git "a/.\\docker-installl-docker-engine.sh" "b/.\\install-docker-engine.sh"
index 5cce019..9479426 100644
--- "a/.\\docker-installl-docker-engine.sh"
+++ "b/.\\install-docker-engine.sh"
@@ -24,8 +24,8 @@ set -e
 #
 
 url="https://get.docker.com/"
-apt_url="https://apt.dockerproject.org"
-yum_url="https://yum.dockerproject.org"
+apt_url="http://mirror.azure.cn/docker-engine/apt"
+yum_url="http://mirror.azure.cn/docker-engine/yum"
 gpg_fingerprint="58118E89F3A912897C070ADBF76221572C52609D"
 
 key_servers="
@@ -239,11 +239,10 @@ do_install() {
 	if [ -z "$lsb_dist" ] && [ -r /etc/oracle-release ]; then
 		lsb_dist='oracleserver'
 	fi
-	if [ -z "$lsb_dist" ] && [ -r /etc/centos-release ]; then
-		lsb_dist='centos'
-	fi
-	if [ -z "$lsb_dist" ] && [ -r /etc/redhat-release ]; then
-		lsb_dist='redhat'
+	if [ -z "$lsb_dist" ]; then
+		if [ -r /etc/centos-release ] || [ -r /etc/redhat-release ]; then
+			lsb_dist='centos'
+		fi
 	fi
 	if [ -z "$lsb_dist" ] && [ -r /etc/os-release ]; then
 		lsb_dist="$(. /etc/os-release && echo "$ID")"
@@ -251,12 +250,6 @@ do_install() {
 
 	lsb_dist="$(echo "$lsb_dist" | tr '[:upper:]' '[:lower:]')"
 
-	# Special case redhatenterpriseserver
-	if [ "${lsb_dist}" = "redhatenterpriseserver" ]; then
-        	# Set it to redhat, it will be changed to centos below anyways
-        	lsb_dist='redhat'
-	fi
-
 	case "$lsb_dist" in
 
 		ubuntu)
@@ -269,7 +262,7 @@ do_install() {
 		;;
 
 		debian)
-			dist_version="$(cat /etc/debian_version | sed 's/\/.*//' | sed 's/\..*//')"
+			dist_version="$(cat /etc/debian_version | sed 's/\/.*//' | sed 's/\..*//' | uniq)"
 			case "$dist_version" in
 				8)
 					dist_version="jessie"
@@ -283,11 +276,11 @@ do_install() {
 		oracleserver)
 			# need to switch lsb_dist to match yum repo URL
 			lsb_dist="oraclelinux"
-			dist_version="$(rpm -q --whatprovides redhat-release --queryformat "%{VERSION}\n" | sed 's/\/.*//' | sed 's/\..*//' | sed 's/Server*//')"
+			dist_version="$(rpm -q --whatprovides redhat-release --queryformat "%{VERSION}\n" | sed 's/\/.*//' | sed 's/\..*//' | sed 's/Server*//' | uniq)"
 		;;
 
-		fedora|centos|redhat)
-			dist_version="$(rpm -q --whatprovides ${lsb_dist}-release --queryformat "%{VERSION}\n" | sed 's/\/.*//' | sed 's/\..*//' | sed 's/Server*//' | sort | tail -1)"
+		fedora|centos)
+			dist_version="$(rpm -q --whatprovides redhat-release --queryformat "%{VERSION}\n" | sed 's/\/.*//' | sed 's/\..*//' | sed 's/Server*//' | uniq)"
 		;;
 
 		*)
@@ -435,18 +428,14 @@ do_install() {
 			done
 			$sh_c "apt-key adv -k ${gpg_fingerprint} >/dev/null"
 			$sh_c "mkdir -p /etc/apt/sources.list.d"
-			$sh_c "echo deb \[arch=$(dpkg --print-architecture)\] ${apt_url}/repo ${lsb_dist}-${dist_version} ${repo} > /etc/apt/sources.list.d/docker.list"
+			$sh_c "echo deb [arch=$(dpkg --print-architecture)] ${apt_url}/repo ${lsb_dist}-${dist_version} ${repo} > /etc/apt/sources.list.d/docker.list"
 			$sh_c 'sleep 3; apt-get update; apt-get install -y -q docker-engine'
 			)
 			echo_docker_as_nonroot
 			exit 0
 			;;
 
-		fedora|centos|redhat|oraclelinux)
-			if [ "${lsb_dist}" = "redhat" ]; then
-				# we use the centos repository for both redhat and centos releases
-				lsb_dist='centos'
-			fi
+		fedora|centos|oraclelinux)
 			$sh_c "cat >/etc/yum.repos.d/docker-${repo}.repo" <<-EOF
 			[docker-${repo}-repo]
 			name=Docker ${repo} Repository
@@ -515,3 +504,4 @@ do_install() {
 # wrapped up in a function so that we have some protection against only getting
 # half the file during "curl | sh"
 do_install
+
```